test.yaml

```diff
@@ -75,3 +75,13 @@ tests:
       users: ['user2']
       base_path: ""
       TLS: False
+     
+output:
+  logfile:
+    logging: True
+  grafana:
+    push : False
+    hostname: ''
+    port: '80'
+    modules: ['Throughput']
```

Comment:
If this restricts the results to push to Grafana, I do not think it is correct.
I think we should push all of them when grafana in enabled